app/presenters/finder_content_item_presenter.rb

```diff
@@ -50,6 +50,8 @@ def details
       summary: metadata.fetch("summary", nil),
       facets: schema.fetch("facets", []),
     }
+    details_hash[:default_order] = metadata["default_order"] if metadata["default_order"]
```

Comment:
I'd be tempted to put this into the main body of the hash (ie, as `default_order: metadata["default_order"]`) and then filter out anything with a nil value in the hash by calling something like `reject {|_, value| value.nil?}` on the hash.

If you have more than one such optional value, this is going to get hard to read quickly.